### ISSUE TITLE
Bob challenge spec doesnt match the tests

### ISSUE THREAD

**Issue by gavinhenderson** [exercism/typescript#260]
The tests for `bob` include the idea of 'forceful questions' which aren't mentioned at all in the spec. Is this by design?

Probably related to #131

**Comment by masters3d** [comment]
Yes. You are welcome to submit a pr if you’ll like for them to match. At some point keeping up with the canonical set was way too much work so we started locking each implementation so they could diverge.
